pset-functions.js

```diff
@@ -136,6 +170,26 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function scrabbleScore(word) {
+    let letters = {
+        "A": 1, "E": 1, "I": 1, "O": 1, "U": 1, "L": 1, "N": 1, "R": 1, "S": 1, "T": 1,
+        "D": 2, "G": 2,
+        "B": 3, "C": 3, "M": 3, "P": 3,
+        "F": 4, "H": 4, "V": 4, "W": 4, "Y": 4,
+        "K": 5,
+        "J": 8, "X": 8,
+        "Q": 10, "Z": 10
+    }
+    wordArray = Array.from(word)
+    score = 0
+    for (letter of wordArray) {
+        upperCaseLetter = letter.toUpperCase()
+        score = score + letters[upperCaseLetter]
+    }
+    console.log(score)
+}
+scrabbleScore("cabbage")
```

Comment:
@laraann88 Good job here on this solution here. I see you created an object of all the letters and their values. Nice job using the for of to loop through each letter and increment the value by accessing it based on the object keys.
Take a look at a solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L174)